tfx/types/artifact_utils_test.py

```diff
@@ -179,6 +179,21 @@ def testIsArtifactVersionOlderThan(self):
     self.assertFalse(
         artifact_utils.is_artifact_version_older_than(examples, '0.1'))
 
+  @mock.patch("tfx.types.artifact_utils.fileio", autospec=True)
```

Comment:
How about adding some test cases for the error case? For example uri is missing or the file doesn't exist.

Reply:
added!